Managers/PresetMan.cpp

```diff
@@ -334,8 +334,9 @@ bool PresetMan::IsModuleUserdata(const std::string &moduleName) const {
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 std::string PresetMan::GetFullModulePath(const std::string &modulePath) const {
-	const std::string modulePathGeneric = std::filesystem::path(modulePath).generic_string();
-	const std::string pathTopDir = modulePathGeneric.substr(0, modulePathGeneric.find_first_of("/\\") + 1);
+	std::string modulePathGeneric = std::filesystem::path(modulePath).generic_string();
+	std::replace(modulePathGeneric.begin(), modulePathGeneric.end(), '\\', '/');
```

Comment:
add a comment explaining the Windows vs non-Windows mod compat stuff. 

Also - is doing it only here enough? Do need to do this in other places too, or do all filepaths go through here?